packages/dart/sshnp_flutter/lib/src/utility/form_validator.dart

```diff
@@ -10,12 +10,23 @@ class FormValidator {
     return null;
   }
 
+  static String? validateRequiredIntField(String? value) {
+    String valid = r'^[0-9]+$';
+    if (value?.isEmpty ?? true) {
+      return kEmptyFieldValidationError;
+    } else if (!RegExp(valid).hasMatch(value!)) {
+      return kIntFieldValidationError;
+    }
+    return null;
+  }
+
```

Comment:
Specifically for ports, we can do one better, and say the range is 0-65535

Reply:
done